stream_alert/rule_processor/handler.py

```diff
@@ -24,6 +27,10 @@
 from stream_alert.rule_processor.rules_engine import StreamRules
 from stream_alert.rule_processor.sink import StreamSink
 from stream_alert.shared.metrics import Metrics
+from stream_alert.shared.stats import time_me
+
+NUM_WORKERS = 6
```

Comment:
Update: just tested this with  1536 MB of memory and it still only gives 2 processors.. :sadparrot:. Testing with 2 workers (obviously) gets us about a 2x speed improvements. I think we should discuss this more and use at least 4 workers.